swig/Version.i.in

```diff
@@ -3,4 +3,9 @@
 #include <pacbio/Version.h>
 %}
 
+%pythoncode
+%{
+__version__ = "@UNANIMITY_VERSION@"
+%}
+
```

Comment:
Actually, this can be __version__ = UnanimityVersion() without every having to configure.